### PR TITLE
Adding Z-indices presets as suggested by @joshwcomeau

### PR DESCRIPTION
**Pull request by mfrachet**
Related comment: https://github.com/gatsby-inc/gatsby-interface/pull/110#pullrequestreview-315645827

Adding z-indices as presets so that we have a clear hierarchy of what can be displayed over what.

### CODE REVIEW

File: src/utils/presets/index.ts [hunk 1]
```diff
@@ -3,3 +3,4 @@ export { fontFamilies, fontSizes } from "./typography"
 export { spaces } from "./spaces"
 export { breakpoints } from "./breakpoints"
 export { dimensions, radius } from "./sizes"
+export * from "./zIndices"
```
Comment (greglobinski) [inline src/utils/presets/index.ts]: Can we explicitly say what we exporting. Just for convenience to see what we exporting without need to dig into another file?

Comment (mfrachet) [reply src/utils/presets/index.ts]: Yep 😊  👍

**Comment by mfrachet** [comment]
Thanks for the catch @greglobinski , I  forget the root index.ts export 😬

**Comment by joshwcomeau** [comment]
Awesome :) thanks for this!

> Thanks for the catch @greglobinski , I always forget the root index.ts export 😬

Yeah, I forget about this as well... I wonder if there's anything we can do tooling-wise to help us remember 🤔 I can't think of anything that wouldn't raise a bunch of false-positives.

**Pull request merged**